Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/validation/TaskValidation.java

```diff
@@ -57,11 +57,15 @@ private boolean checkUpdateNameDoesNotExist(Long taskId, String name) {
     }
 
     public ValidationResult isDurationValid(String duration) {
-        ValidationResult validationResult = isStringEmpty(duration);
-        if (validationResult.getValidationStatus().equals(ValidationStatus.INVALID)) {
+        if (duration.isEmpty()) {
+            return new ValidationResult(ValidationStatus.VALID);
+        }
+
+        ValidationResult validationResult = isNumber(duration);
```

Comment:
Looks really good! Thanks! 👍  If I could ask you to make a small update.. Lines 64-69 are already covered in isNumber function. And it would be great to not repeat it. Could you please change these line to 
`return isNumber(duration) 
`only?

Reply:
Oh, that's dumb of me! Sure, I am on it.